app/server/api/auth_api.py

```diff
@@ -624,7 +624,7 @@ def post(self):
             if not user:
                 response_object = {
                     'status': 'fail',
-                    'message': 'User not found'
+                    'message': 'invalid password'
                 }
```

Comment:
This still doesn't fix the issue. See [penetration testing notes](https://drive.google.com/file/d/1e_lD3RzRcii2hbNy_yTwAneSUk40FG6S/view?usp=sharing) and https://github.com/teamsempo/SempoBlockchain/blob/133aaab34f2b4c214d0c57bfb7a1b5abaf1b0426/app/server/api/auth_api.py#L575-L589

Reply:
D'oh! Good catch @tristanhcole! Added the new error on ln578 too. Think rate limit would be out of scope and for another PR?

Reply:
Yep, makes sense to have rate limiting in separate PR

Reply:
Implemented in #703